ctfhub/models.py

```diff
@@ -119,7 +119,7 @@ class VisibilityType(models.TextChoices):
     visibility = models.CharField(
         max_length=4, choices=VisibilityType.choices, default=VisibilityType.PUBLIC
     )
-    weight = models.FloatField(default=1.0, blank=False)
+    weight = models.FloatField(default=0.0, blank=False)
```

Comment:
I'm not super convinced that the default should be 0. You would want to a default to be something not null. Care to justify?

Reply:
NVM I hadn't read your comment